core/state-machine/src/ext.rs

```diff
@@ -120,6 +123,7 @@ where
 			changes_trie_transaction: None,
 			offchain_externalities,
 			keystore,
+			id: rand::random(),
```

Comment:
Any ways to use a sequential index instead, or to lazyly init it behind a refcell?

Reply:
I can do a pr for it if it sounds usefull (refcell).

Reply:
I thought about that but decided that random is good enough for debugging. Collision chance is rather small. For sequential counter I'd probably use `AtomicU16::fetch_add`

Reply:
Oh yes, atomic u16 seems better than rand to me for any consideration, and probably not worth lazy loading with a refcell :+1: 
(that was more the call to rand than the collision possibility that was bugging me).